README.md

```diff
@@ -1,7 +1,9 @@
-Orleans - Distributed Actor Model
+<p align="center">
+  <img src="https://github.com/dotnet/orleans/blob/gh-pages/assets/logo_full.png" alt="Orleans logo" width="600px"> 
+</p>
```

Comment:
No, I centering of the badge isn't in this PR, wanted to hear opinions about it before of the including into PR.